compiler/passes/src/type_checking/check_expressions.rs

```diff
@@ -574,7 +633,7 @@ impl<'a> ExpressionVisitor<'a> for TypeChecker<'a> {
                 if let Some(func) = func {
                     // Check that the call is valid.
```

Comment:
@d0cd Main portions to review in TYC: visit_call, visit_access, visit_return, visit_definition. visit_stub, visit_program_scope secondary